examples/Makefile

```diff
@@ -724,3 +724,16 @@ get-java-deps:
 get-java-deps-verify:
 	kubeless function call get-java-deps --data '{"hello": "world"}'
 	kubectl logs -l function=get-java-deps | grep -q '.*Hello.*world! Current local time is:'
+
+get-julia:
```

Comment:
you also need to add these examples as tests in `tests/integration-tests.bats`

Reply:
OK

Reply:
get it.